src/rgw/driver/d4n/rgw_sal_d4n.cc

```diff
@@ -2576,7 +2546,7 @@ int D4NFilterMultipartUpload::complete(const DoutPrefixProvider *dpp,
   rgw::sal::Attrs attrs;
   d4n_target_obj->load_obj_state(dpp, y);
   attrs = d4n_target_obj->get_attrs();
-  d4n_target_obj->set_attrs_from_obj_state(dpp, y, attrs);
+  d4n_target_obj->set_attrs_from_obj_state(dpp, y, attrs, false); // TODO: is this value correct?
```

Comment:
What should the `dirty` value be here?

Reply:
For multipart uploads it is always clean/non-dirty